be/src/vec/core/block.cpp

```diff
@@ -689,15 +690,16 @@ Status Block::filter_block(Block* block, int filter_column_id, int column_to_kee
     return Status::OK();
 }
 
-Status Block::serialize(PBlock* pblock, size_t* uncompressed_bytes, size_t* compressed_bytes,
+/*Status Block::serialize(PBlock* pblock, size_t* uncompressed_bytes, size_t* compressed_bytes,
```

Comment:
Maybe we can delete those code directly.

Reply:
done